atlas/atlas.properties

```diff
@@ -0,0 +1,15 @@
+# PostGreSQL
+psql.url=jdbc:postgresql://localhost:5432/georchestra
+psql.user=www-data
+psql.pass=www-data
+
+# SMTP configuration
+smtpHost=localhost
+smtpPort=25
+
+# Other
+atlas.baseUrl=http://localhost:8080/atlas
```

Comment:
https://georchestra.mydomain.org/atlas to match what is done elsewhere in the current datadir